api/v1alpha1/nodehealthcheck_types.go

```diff
@@ -167,51 +179,58 @@ type EscalatingRemediation struct {
 
 // NodeHealthCheckStatus defines the observed state of NodeHealthCheck
 type NodeHealthCheckStatus struct {
-	// +operator-sdk:csv:customresourcedefinitions:type=status,displayName="observedNodes",xDescriptors="urn:alm:descriptor:com.tectonic.ui:observedNodes"
-	//ObservedNodes specified the number of nodes observed by using the NHC spec.selector
-	// +kubebuilder:default:=0
+	// ObservedNodes specified the number of nodes observed by using the NHC spec.selector
+	//
+	//+kubebuilder:default:=0
+	//+operator-sdk:csv:customresourcedefinitions:type=status
```

Comment:
Nit: I've noticed that `xDescriptors` and `displayName` were removed, just want to make sure that is intended.

Reply:
intended:
- `urn:alm:descriptor:com.tectonic.ui:observedNodes` isn't a valid descriptor
- display name is optional, and "observedNodes" isn't a nice display name